public-ui/src/app/app-routing.module.ts

```diff
@@ -51,7 +51,7 @@ const routes: Routes = [
         data: {title: 'View Full Results'}
       },
       {
-        path: 'physical-measurements',
+        path: 'physical-measurements/:searchString',
```

Comment:
It's a bit more typical to use a query parameter for this kind of thing - you can subscribe to query parameters in the same way you're doing for normal parameters.

e.g. `/physical-measurements?q=asdf`

Reply:
Changed the param to query param.